docs/update_requirememts.md

```diff
@@ -0,0 +1,12 @@
+# Updating python packages and their versions 
```

Comment:
I have updated the devctr-image.md, as how the requirements-freeze.txt get updated. 
I still want to keep this doc file because this script could be executed outside of the devctr-image.md. Let's says you don't want to build a new docker images but just want a copy of requirements-freeze.txt from container (given you have an image id) you can use this script.

Reply:
if we keep this file, maybe just link to it from the `devctr-image.md` file.

Reply:
sounds good. done.